pkg/service/volume_service_test.go

```diff
@@ -24,7 +25,7 @@ type VolumeServiceTestSuite struct {
 
 func (suite *VolumeServiceTestSuite) config() *NsGroupService {
 
-	groupService, err := NewNsGroupService("10.18.174.8", "admin", "admin", "v1", true)
+	groupService, err := NewNsGroupService("10.18.174.8", "admin", "admina", "v1", true)
```

Comment:
Do not include a password.  One option is to require an environment variable to be set when the test is run.

Reply:
It is not okay.  Email me if you would like to discuss it.

Reply:
removed it